dietpi/dietpi-software

```diff
@@ -4612,7 +4627,7 @@ The install script will now exit. After applying one of the the above, rerun die
 					fi
 
 				}
-				G_EXEC ncc maintenance:install --no-interaction --database 'mysql' --database-name 'nextcloud' --database-user 'tmp_root' --database-pass "${nc_password//?/X}" --admin-user "$username" --admin-pass "${GLOBAL_PW//?/X}" --data-dir "$datadir"
+				G_EXEC ncc maintenance:install --no-interaction --database='mysql' --database-name='nextcloud' --database-user='tmp_root' --database-pass="${nc_password//?/X}" --admin-user="$username" --admin-pass="${GLOBAL_PW//?/X}" --data-dir="$datadir"
```

Comment:
Are you sure that this is correct to add =? At least following is suggested by the error message.

```
maintenance:install [--database DATABASE] [--database-name DATABASE-NAME] [--database-host DATABASE-HOST]         
[--database-port DATABASE-PORT] [--database-user DATABASE-USER] [--database-pass [DATABASE-PASS]]                 
[--database-table-space [DATABASE-TABLE-SPACE]] [--admin-user ADMIN-USER] [--admin-pass ADMIN-PASS] [--admin-email
[ADMIN-EMAIL]] [--data-dir DATA-DIR]
```

Reply:
Can you retry with latest commit? Should now be fixed.

Reply:
Yes way better now 👍